recipes-wpe/wpewebkit/wpewebkit_0.1.bb

```diff
@@ -39,7 +39,7 @@ PROVISIONING_x86 = ""
 PROVISIONING_hikey-32 = ""
 PROVISIONING_dragonboard-410c-32 = ""
 
-PACKAGECONFIG ?= "2dcanvas deviceorientation fullscreenapi encryptedmediav1 fetchapi gamepad geolocation indexeddb libinput logs mediasource notifications nativevideo sampling-profiler shadowdom subtitle subtlecrypto udev video webaudio ${WPE_PLATFORM}"
+PACKAGECONFIG ?= "2dcanvas deviceorientation fullscreenapi encryptedmediav1 fetchapi gamepad geolocation indexeddb libinput logs mediasource notifications nativevideo sampling-profiler shadowdom subtitle subtlecrypto touch udev video webaudio ${WPE_PLATFORM}"
```

Comment:
Oh ok. I saw `deviceorientation` and  `geolocation` which also is pretty rare and thought that the default PACKAGECONFIG was adding most features as a showcase and let downstream set the exact configs they want.